Sources/Widgets/Representations/RectangleContextRepresentation/index.js

```diff
@@ -59,53 +59,35 @@ function vtkRectangleContextRepresentation(publicAPI, model) {
     const dataset = vtkPolyData.newInstance();
 
     if (state.getVisible()) {
-      const bounds = state.getBounds();
-      const point1 = [bounds[0], bounds[2], bounds[4]];
-      const point2 = [bounds[1], bounds[3], bounds[5]];
+      const normal = state.getDirection();
+      const vertical = state.getUp();
+      const horizontal = state.getRight();
+      vec3.cross(horizontal, vertical, normal);
```

Comment:
I find it confusing that those vectors variable names are different (up/vertical, right/horizontal, normal/orient/direction)

Reply:
I didn't like up and right because they could be down or left for that matter, so I used vertical and horizontal... I'll go back to up and right though since that shows the relative coordinate system